### ISSUE TITLE
Add snapshot test in Github action workflow

### ISSUE THREAD

**Issue by kushaggarwal** [creativecommons/fonts#72]
**Description**

This issue aims at the addition of the snapshot test in the current Github workflow

**Problems solved**

Snapshot test added in the current Github workflow currently comprising of only the unit test

**Alternatives**

**Additional context**

**Want to take up?**

Yes

**Comment by megha070** [comment]
Hey @kushaggarwal I have already opened this issue #65. This has been blocked by @dhruvkb until all of the issues are rectified in vocabulary for running visual regression tests.

**Comment by kushaggarwal** [comment]
Okay sure

**Comment by dhruvkb** [comment]
Closing as a duplicate of #65, rightly pointed out by @megha070.
